src/Connector.js

```diff
@@ -47,6 +53,7 @@ class Connector {
   async clearListeners(thingId) {
     await this.client.unsubscribe(`device.${thingId}.data.request`);
     await this.client.unsubscribe(`device.${thingId}.data.update`);
+    await this.client.unsubscribe('device.config.updated');
```

Comment:
I think if one thing was removed, then the connector won't be able to listen this event from another things. It's better to ensure that `this.devices` is and empty list (`=== []`) before calling the unsubscription.

Reply:
Agree with you. Another approach is to avoid unsubscribing from this event since it will be useful in the long-running operation of the connector.

Reply:
Done.
I preferred to remove this unsubscription. What do you think?

Reply:
Agreed!